### PR TITLE
Broken site reporting reference test

### PR DESCRIPTION
**Pull request by Bunn**
Task/Issue URL: https://app.asana.com/0/0/1203476103622317/f

**Description**:
Add broken site reporting reference tests

**Steps to test this PR**:
1. Run tests, see if they work
2. Report broken site and check if it's still working (adding a breakpoint into the request at the send method on BrokenSiteInfo to check the params should be good enough

**Device Testing**:
* [] iPhone 14 Pro
* [] iPad

**OS Testing**:

* [] iOS 14
* [] iOS 15
* [] iOS 16

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

**Comment by jaceklyp** [comment]
qq: wasn't it possible to move broken-site-test into bsk?

File: DuckDuckGoTests/BrokenSiteReportingTests.swift [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    let exceptPlatforms: [String]
+    let manufacturer, model, os: String?
+    let gpcEnabled: Bool?
+}
+
+// MARK: - ExpectReportURLParam
+
+private struct ExpectReportURLParam: Codable {
+    let name, value: String
+}
```
Comment (jaceklyp) [inline DuckDuckGoTests/BrokenSiteReportingTests.swift:154]: could you please move the file under UnitTests/Core/PrivacyReferenceTests?

Comment (jaceklyp) [reply DuckDuckGoTests/BrokenSiteReportingTests.swift:154]: other than that, lgtm!

**Comment by Bunn** [comment]
> qq: wasn't it possible to move broken-site-test into bsk?

Possible? Yes!
But the reporting has different parameters depending on the platform, and BSK should be platform agnostic. More details [here](https://app.asana.com/0/0/1203426104407935/1203438281999446/f)

**Pull request merged**